src/main/java/io/logz/logback/LogzioSender.java

```diff
@@ -67,11 +67,11 @@ private LogzioSender(String logzioToken, String logzioType, int drainTimeout, in
                         boolean addHostname, String additionalFields, int gcPersistedQueueFilesIntervalSeconds)
             throws IllegalArgumentException {
 
-        this.logzioToken = logzioToken;
+        this.logzioToken = getValueFromSystemEnvironmentIfNeeded(logzioToken);
         this.logzioType = logzioType;
         this.drainTimeout = drainTimeout;
         this.fsPercentThreshold = fsPercentThreshold;
-        this.logzioUrl = logzioUrl;
+        this.logzioUrl = getValueFromSystemEnvironmentIfNeeded(logzioUrl);
```

Comment:
👍 
We have this logic also in additionalFields. Can you please re-use this function there as well?